.github/workflows/release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         include:
           - os: ubuntu-latest
             target: x86_64-unknown-linux-musl
-            flags: --no-default-features --features terminology
+            flags: --no-default-features --features terminology,kitty,remote_resources
```

Comment:
Thanks :)  Did you check the `ci.yml` workflow if there's a similar build where you could add `kitty`, so that it gets tested regularly and not just on a release?  Sorry for asking this; I don't remember which builds I set up in `ci.yml` 😇

Reply:
I've checked that, too. I just added the kitty feature to the test_linux job.